extensions/async-extensions/build.gradle

```diff
@@ -8,6 +8,15 @@ plugins {
 archivesBaseName = 'sqldelight-async-extensions'
 
 kotlin {
+  applyDefaultHierarchyTemplate {
+    it.common {
+      it.group("sync") {
```

Comment:
Coroutines calls this source set "concurrent". Should we mirror that?

Reply:
Good idea